.azure/gpu-tests-pytorch.yml

```diff
@@ -21,6 +21,7 @@ pr:
   paths:
     include:
       - ".azure/gpu-tests-pytorch.yml"
+      - "tests/tests_pytorch/run_standalone_*.sh"
```

Comment:
```suggestion
```
The addition here overlaps with the below line, no?
```yml
      - "tests/tests_pytorch/**"
```

Reply:
Good catch. I'll remove it